Idea.js

```diff
@@ -1,9 +1,9 @@
 class Idea {
-  constructor(title, body) {
-    this.id = Date.now();
+  constructor(title, body, star) { //IMPORTANT NOTE: Adding id to our params changes the order in which these display on the card causing an undefined
+    this.id = Date.now() || id; //when these two are switched the console throws an error. Not sure why we need the "or", can we talk about this?
```

Comment:
We need this to be "id || Date.now()" so that if there is an argument passed into the id parameter, it will be used, otherwise, a new id will be assigned.